app/cdap/components/WrangleHome/Components/OngoingDataExplorations/index.tsx

```diff
@@ -141,8 +154,15 @@ export default function() {
               const totalDataQuality = dataQuality / workspace.headers?.length ?? 1;
               explorationData[index].dataQuality = totalDataQuality;
               explorationData[index].count = workspace.count;
-              const final = getUpdatedExplorationCards(explorationData);
+              const final = getUpdatedExplorationCards(explorationData, cardCount);
+              /**
+               * if we have setData, then we should send data to parent element as the exploration state is then maintained in parent as well for showing or hiding the title of the parent component
+               */
+              if (setShowExplorations) {
+                setShowExplorations(final && Array.isArray(final) && final.length ? true : false);
```

Comment:
can we do setShowExplorations && setShowExplorations(final && Array.isArray(final) && final.length ? true : false);